src/julia-parser.scm

```diff
@@ -68,7 +88,9 @@
                              (pushprec (cdr L) (+ prec 1)))))
                      (pushprec (map eval prec-names) 1)
                      t))
-(define (operator-precedence op) (get prec-table op 0))
+(define (operator-precedence op) (get prec-table
+                                      (maybe-strip-op-suffix op)
```

Comment:
Testing for operators can be pretty important for performance. It would be nice to only call `maybe-strip-op-suffix` for precedence levels that support it.